BlogTemplate.Tests/Models/BlogDataStoreTests.cs

```diff
@@ -112,48 +112,48 @@ public void CreateSlug_ReturnValidSlug()
             Post test = new Post
             {
                 Title = "Test Title",
-                Slug = testSlug.CreateSlug("Test Title"),
                 Body = "Test body",
                 PubDate = DateTime.Now,
                 LastModified = DateTime.Now,
                 IsPublic = true,
                 Excerpt = "Test excerpt",
             };
+            test.Slug = testSlug.CreateSlug(test);
             testDataStore.SavePost(test);
 
 
             Post test1 = new Post
             {
                 Title = "Test Title",
-                Slug = testSlug.CreateSlug("Test Title"),
                 Body = "Test body",
                 PubDate = DateTime.Now,
                 LastModified = DateTime.Now,
                 IsPublic = true,
                 Excerpt = "Test excerpt",
             };
+            test1.Slug = testSlug.CreateSlug(test1);
             testDataStore.SavePost(test1);
 
             Post test2 = new Post
             {
                 Title = "Test Title",
-                Slug = testSlug.CreateSlug("Test Title"),
                 Body = "Test body",
                 PubDate = DateTime.Now,
                 LastModified = DateTime.Now,
                 IsPublic = true,
                 Excerpt = "Test excerpt",
             };
+            test2.Slug = testSlug.CreateSlug(test2);
             testDataStore.SavePost(test2);
 
-            Post result = testDataStore.GetPost("Test-Title");
-            Assert.Equal("Test-Title", result.Slug);
+            Post result = testDataStore.GetPost($"{test.PubDate.ToFileTimeUtc()}_Test-Title");
+            Assert.Equal($"{test.PubDate.ToFileTimeUtc()}_Test-Title", result.Slug);
```

Comment:
Seems like the slug should still be Test-Title, and should be separate from the filename <timestamp>_Test-Title.